pkg/interface/src/logic/subscription/global.ts

```diff
@@ -43,18 +43,27 @@ export default class GlobalSubscription extends BaseSubscription<StoreState> {
   };
 
   start() {
-    this.subscribe('/all', 'invite-store');
-    this.subscribe('/groups', 'group-store');
-    this.subscribe('/primary', 'contact-view');
     this.subscribe('/all', 'metadata-store');
-    this.subscribe('/all', 's3-store');
-    this.subscribe('/all', 'launch');
-    this.subscribe('/all', 'weather');
-    this.subscribe('/keys', 'graph-store');
-    this.subscribe('/updates', 'hark-store');
-    this.subscribe('/updates', 'hark-graph-hook');
-    this.subscribe('/updates', 'hark-group-hook');
-    this.subscribe('/updates', 'hark-chat-hook');
+    this.subscribe('/all', 'invite-store', true);
```

Comment:
If you want this to go in the same batch, shouldn't the `metadata-store` subscription have the `queue` flag set also?

Reply:
Why do we even need to batch the requests if they're in a queue?